cmd/chart/upgrade.go

```diff
@@ -200,6 +200,13 @@ func updateImages(iName string, v bool) (bool, string, error) {
 
 func tagIsUpgradeable(currentTag, latestTag string) bool {
 
-	return latestTag != currentTag && !strings.Contains(strings.ToLower(latestTag), "-rc") && !strings.EqualFold(currentTag, "latest")
+	if strings.EqualFold(currentTag, "latest") {
+		return false
+	}
+
+	currentSemVer, _ := semver.NewVersion(currentTag)
+	latestSemVer, _ := semver.NewVersion(latestTag)
+
+	return latestTag != currentTag && len(latestSemVer.Prerelease()) == 0 && len(currentSemVer.Prerelease()) == 0
```

Comment:
What about the case where the value is `v0.15.1-rootless` and we want to go up to `v0.15.2-rootless`?

Reply:
That will not upgrade.  

To achieve that we would need to check that the current/latest prereleases match.